### DISCUSSION TITLE
Logto for flutter web

### DISCUSSION THREAD

**Discussion by RajaShanmugamJM** [logto-io/logto#5505] [Q&A]
Hey there,

I'm trying to integrate logto into flutter app and when I tried to run flutter app on web, it seems the `logto_dart_sdk` is not supported for web or am I wrong?

<img width="538" alt="image" src="https://github.com/logto-io/logto/assets/27273337/9a61ddfe-7976-46c7-ba75-ea74a93e6800">

<img width="762" alt="image" src="https://github.com/logto-io/logto/assets/27273337/47a9b4d5-88bf-455c-b8a1-6b143a4df87e">

**Comment by simeng-li** [comment]
Correct. The SDK is suitable for Android and iOS platforms only. The redirect URI format is restricted to the native scheme.

**Reply by jonsamwell** [reply]
Is there any plans to support this or do you suggest going down the vanillajs route for Flutter web apps?

**Reply by louis-sicko** [reply]
> Is there any plans to support this or do you suggest going down the vanillajs route for Flutter web apps?

I also would love to have this.
